README.md

```diff
@@ -57,7 +57,7 @@ See [Contributing](.github/CONTRIBUTING.md).
   - [File Transfer - Web-based File Managers](#file-transfer---web-based-file-managers)
   - [Games](#games)
   - [Games - Administrative Utilities & Control Panels](#games---administrative-utilities--control-panels)
-  - [Gateways and Terminal Sharing](#gateways-and-terminal-sharing)
+  - [Gateways and Sharing](#gateways-and-terminal-sharing)
```

Comment:
please undo this change

Reply:
Hi, I'm confused by the context here. Do you want line 60 to read
```suggestion
  - [Gateways and Sharing](#gateways-and-terminal-sharing)
```
or 
```suggestion
  - [Gateways and Terminal Sharing](#gateways-and-terminal-sharing)
```
Apologies if the answer is obvious, but I can't seem to reason out what you want 🙈

Reply:
Gateway and Terminal Sharing as it was before the change